contrib/runners/winrm_runner/winrm_runner/runner.yaml

```diff
@@ -0,0 +1,133 @@
+- description: A remote execution runner that executes PowerShell commands via WinRM on a set of remote hosts
+  enabled: true
+  name: winrm-ps-cmd
+  runner_package: winrm_runner
+  runner_module: winrm_ps_command_runner
+  runner_parameters:
+    cmd:
+      description: Arbitrary PowerShell command to be executed on the remote host(s).
+      type: string
+    cwd:
+      description: Working directory where the command will be executed in
+      type: string
+    env:
+      description: Environment variables which will be available to the command (e.g.
+        key1=val1,key2=val2)
+      type: object
+    host:
+      description: A host where the command will be run
+      required: true
+      type: string
+    kwarg_op:
+      default: -
+      description: Operator to use in front of keyword args i.e. "-" or "/".
+      type: string
+    password:
+      description: Password used to log in.
+      required: true
+      secret: true
+      type: string
+    port:
+      default: 5986
+      description: 'WinRM port to connect on. If using port 5985 scheme must be "http"'
+      required: false
+      type: integer
+    scheme:
+      default: "https"
+      description: 'Scheme to use in the WinRM URL. If using scheme "http" port must be 5985'
+      required: false
+      type: string
+    timeout:
+      default: 60
+      description: Action timeout in seconds. Action will get killed if it doesn't
+        finish in timeout seconds.
+      type: integer
+    transport:
+      default: "ntlm"
+      description: The type of transport that WinRM will use to communicate.
+        See https://github.com/diyan/pywinrm#valid-transport-options
+      required: false
+      type: string
+      enum:
+        - "basic"
+        - "certificate"
+        - "credssp"
+        - "kerberos"
+        - "ntlm"
+        - "plaintext"
+        - "ssl"
+    username:
+      description: Username used to log-in.
+      required: true
+      type: string
+    verify_ssl_cert:
+      default: true
+      description: Certificate for HTTPS request is verified by default using requests
+        CA bundle which comes from Mozilla. Verification using a custom CA bundle
+        is not yet supported. Set to False to skip verification.
+      type: boolean
+- description: A remote execution runner that executes PowerShell script via WinRM on a set of remote hosts
+  enabled: true
+  name: winrm-ps-script
+  runner_package: winrm_runner
+  runner_module: winrm_ps_script_runner
+  runner_parameters:
+    cwd:
+      description: Working directory where the command will be executed in
+      type: string
+    env:
+      description: Environment variables which will be available to the command (e.g.
+        key1=val1,key2=val2)
+      type: object
+    host:
+      description: A host where the command will be run
+      required: true
+      type: string
+    kwarg_op:
+      default: -
```

Comment:
Looks like this may need to be quoted, otherwise the runner fails to register with `yaml.scanner.ScannerError: sequence entries are not allowed here`

Seems to allow `--` without quoting, but not `-`.